Update info for PostgreSQL version 15

rhea-sys opened the pull request:
There was a new release for PostgreSQL (version 15). I replaced '14' by '15' in the Windows, Mac and Linux setup files in the installation commands.

karlhorky commented on linux.md:
```diff
@@ -176,7 +176,7 @@
     wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
     sudo apt update
-    sudo apt install postgresql-14
+    sudo apt install postgresql-15
```
Hm... is `postgres-15` really an option on the Debian Apt package?

> Currently, we support
> ...
> - PostgreSQL 10, 11, 12, 13, 14, [15 beta](https://wiki.postgresql.org/wiki/Apt/FAQ#I_want_to_try_the_beta_version_of_the_next_PostgreSQL_release), [16 devel](https://wiki.postgresql.org/wiki/Apt/FAQ#Development_snapshots)

Ref: https://wiki.postgresql.org/wiki/Apt

karlhorky replied:
This blog post (from Oct. 17) seems to think that you can use it... https://www.linuxtuto.com/how-to-install-postgresql-15-on-debian-11/

karlhorky replied:
Another blog post here (also Oct 17): https://www.huuii.com/site/hacking/3655

karlhorky replied:
There are some other posts that mention this as well. For Linux support, maybe this is enough, since we'll probably have first Linux users following this guide only starting in the next months.

karlhorky commented on macos.md:
````diff
@@ -151,7 +151,7 @@
 
     ```bash
-    [[ -d /opt/homebrew/var/postgresql@14 ]] && PGDATA_TMP=/opt/homebrew/var/postgresql@14 || PGDATA_TMP=/usr/local/var/postgres
+    [[ -d /opt/homebrew/var/postgresql@15 ]] && PGDATA_TMP=/opt/homebrew/var/postgresql@14 || PGDATA_TMP=/usr/local/var/postgres
````
looks like we have another `14` on this line

rhea-sys commented:
I updated the '14' to '15', that I had forgotten in macos.md and changed the linux.md changes back, because it is not officially supported yet. Is it correct like this?

karlhorky commented:
LGTM, thanks! 🚀
